ts/Grid/Core/Options.ts

```diff
@@ -524,6 +524,11 @@ export interface CreditsOptions {
      */
     text?: string;
```

Comment:
Update the default

Reply:
Fixed, but the final definition will be added when we agree about path of the icon.